### PR TITLE
Disabled property for FinalForm inputs

### PR DESCRIPTION
**Pull request by Hyperkid123**
Added disabled prop and missing styling for FinalFormFields.

### CODE REVIEW

File: src/forms/style.scss [hunk 1]
```diff
@@ -60,6 +60,27 @@ $input-default-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
   }
 }
 
+.final-form-select.is-disabled {
+  .Select-control {
```
Comment (ohadlevy) [inline src/forms/style.scss]: just wondering, why do we need special css? isnt it by default by bootstrap/patternfly when the disable attribute is set?

Comment (Hyperkid123) [reply src/forms/style.scss]: apparently no. Adding just the disabled prop has no effect on the component.

**Comment by ohadlevy** [comment]
I wonder if we are missing anything? as when we did the same for
redux-forms there was no need to add additional css? (nor do i think i had
to do it before)...

**Comment by Hyperkid123** [comment]
@ohadlevy what component did u use? This implementation uses ReactSelect. When i was creating these components there was no select component in patternfly-react.

**Comment by Hyperkid123** [comment]
@karelhala

**Comment by Hyperkid123** [comment]
@miq-bot assign @karelhala

**Pull request merged**